code/week2/operations.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+#include <iostream>
+
+int add(int a, int b);
```

Comment:
Header files usually have method documentation, maybe this file should have docs?